src/components/Services/DtsList.tsx

```diff
@@ -14,7 +14,10 @@ import { Configuration, ConfigurationParameters } from "../../openapi-client";
 import { useAuth } from "react-oidc-context";
 import { Log } from "oidc-client-ts";
 import Link from "next/link";
-import Pagination from "../Pagination/Pagination";
+import Pagination from "../../app/ui/Pagination/Pagination";
+import { GenericEntityResourceApi, GenericStateEntityTypeIdNewStatePutRequest } from '../../openapi-client/apis/GenericEntityResourceApi';
+import WarningTimedToast from '../../app/ui/TimedToasts/WarningTimedToast';
```

Comment:
please add relative path, in ts the relative path is based on index.ts file with the resume of routes